.github/workflows/ci.yml

```diff
@@ -296,4 +296,4 @@ jobs:
           tag_name=${GITHUB_REF#refs/tags/}
           hub release create -m $tag_name $tag_name $(for f in cachepot-*.tar.gz*; do echo "-a $f"; done)
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
micro nitpick: let's keep the trailing EOL

Reply:
Those whitespaces are getting the best of me :upside_down_face: